client/app/cloud/project/compute/infrastructure/privateNetwork/addVRack/cloud-project-compute-infrastructure-privateNetwork-addVRack.html

```diff
@@ -0,0 +1,60 @@
+<div class="oui-modal oui-modal_shadow">
+    <div class="oui-modal__header">
+        <button class="oui-icon oui-icon-close_thin oui-modal__close-button" type="button" aria-label="Exit"
+                data-ng-click="AddVRackCtrl.cancel()"></button>
+    </div>
+    <div class="oui-modal__body oui-modal__body_no-icon">
+        <form class="oui-modal__body-text" id="vRackActivateForm" name="vRackActivateForm" novalidate>
+            <h3 class="oui-modal__title activate-private-network-modal-header" data-ng-bind="'private_network_activate_title' | translate"></h3>
+            <div class="radio activate-private-network-radio">
+                <label class="full-width bottom-space-m8" data-ovh-stop-event="click" for="existingVrack">
+                    <input name="activateOptions" id="existingVrack"
+                        class="no-box-shadow vertical-text-bottom"
+                        data-ovh-stop-event="click"
+                        type="radio"
+                        data-ng-value="AddVRackCtrl.activateOptions.EXISTING"
+                        data-ng-model="AddVRackCtrl.selectedActivateOption">
+                    {{ :: "private_network_activate_existing" | translate }}
+                    <div class="activate-private-network-vRack-list">
+                        <div class="oui-select activate-private-network-select">
+                            <select class="oui-select__input"
+                                id="vRacks" name="vRacks"
+                                data-ng-model="AddVRackCtrl.selectedVrack"
+                                data-ng-options="vRack.displayName for vRack in AddVRackCtrl.vRacks"
+                                required />
+                            <i class="oui-icon oui-icon-chevron-down"></i>
+                        </div>
+                    </div>
+                </label>
+            </div>
+
+            <div class="radio activate-private-network-radio">
```

Comment:
I think your radio should look like this => 
http://master.ui-kit.ovh/#!/ovh-ui-kit/radio

See under Usage => Description.  No custom css class should be needed (or very little)

Reply:
fixed